core/runtime/Cargo.toml

```
@@ -14,7 +14,12 @@ rust-version.workspace = true
 boa_engine.workspace = true
 boa_gc.workspace = true
 boa_interop.workspace = true
+either.workspace = true
+futures-lite = { workspace = true, optional = true }
+http = { version = "1.3.1", optional = true }
+reqwest = { version = "0.12.20", optional = true }
```

Nitpick: Consider putting theses in workspace dependencies, so we have one place where all versions are defined, avoiding potential compilation issues related to different versions
```suggestion
http = { version = "1.3.1", optional = true }
reqwest = { version = "0.12.20", optional = true }
```